src/_guides/language/effective-dart/usage.md

```diff
@@ -117,6 +116,13 @@ Instead, when you need to reach into a package's `lib` directory
 (even from the same package's `test` directory
 or any other top-level directory),
 use a `package:` import.
+
+{:.good}
+<?code-excerpt "test/api_test.dart"?>
```

Comment:
We'll need to change the excerpt tag and the test/api_test.dart file to make the build work, but I can handle that.